source/simulator/core.cc

```diff
@@ -797,23 +797,6 @@ namespace aspect
                 break;
               }
           }
-#else
```

Comment:
You are removing both the `#else` and the `#endif`. You will then also have to remove the `#if` above.

Reply:
Removed.